packet/conn.go

```diff
@@ -310,6 +338,16 @@ func (c *Conn) WritePacket(data []byte) error {
 	return nil
 }
 
+func (c *Conn) writeWithTimeout(b []byte) (n int, err error) {
```

Comment:
in line 415, the `c.Write` should also be replace by this function

Reply:
done